packages/common/infra/src/atom/settings.ts

```diff
@@ -1,5 +1,9 @@
+import { setupGlobal } from '@affine/env/global';
 import { atom } from 'jotai';
 import { atomWithStorage } from 'jotai/utils';
+import { atomEffect } from 'jotai-effect';
+
+setupGlobal();
```

Comment:
why call setup here? not in a app start progress.

Reply:
Due to some module import side effect sequence issue, there is a case that environment variables have not being initialized yet.

Reply:
ok, but leave some comments to improve those codes?

Reply:
Seems those sequence issue should be resolved.